CMakeLists.txt

```diff
@@ -345,6 +346,22 @@ else (HAVE_USING_OPERATOR)
   set (ac_cv_cxx_using_operator 0)
 endif (HAVE_USING_OPERATOR)
 
+if (HAVE_EXECINFO_H)
+  set (HAVE_STACKTRACE 1)
+endif (HAVE_EXECINFO_H)
+
+if (WIN32)
+  set (HAVE_STACKTRACE 1)
```

Comment:
As long as code is getting corrected, let's delete [this unnecessary include](https://github.com/google/glog/blob/5cfa9a28f51c41bcafd993f6e93a28daf977d276/src/stacktrace_windows-inl.h#L37) as `CaptureStackBackTrace` comes from the Windows [kernel](https://msdn.microsoft.com/en-us/library/windows/desktop/bb204633(v=vs.85).aspx).